src/elexmodel/utils/math_utils.py

```diff
@@ -30,7 +34,16 @@ def weighted_median(x, weights):
 
     # find index of largest x_i where weights are less than or equal 0.5
     weights_cumulative = np.cumsum(weights_sorted)
-    median_index = np.where(weights_cumulative <= 0.5)[0][-1]
+    # conformity scores are lined up in size order, but each is assigned a
```

Comment:
this function is more general, so could you change the comment to just be about weights?

Reply:
changed to general 'x-value'